pcsx2/gui/MainFrame.cpp

```diff
@@ -643,31 +643,65 @@ void MainEmuFrame::ApplyCoreStatus()
 		}
 	}
 
-	if( cdvd )
+	if (cdvd2 == nullptr) // It doesn't point to anything when it's removed , let's restore it.
 	{
-		if( vm )
-		{
-			cdvd->SetItemLabel(_("Reboot CDVD (full)"));
-			cdvd->SetHelp(_("Hard reset of the active VM."));
-		}
-		else
-		{
-			cdvd->SetItemLabel(_("Boot CDVD (full)"));
-			cdvd->SetHelp(_("Boot the VM using the current DVD or Iso source media"));
-		}	
+		m_menuSys.Prepend(cdvd2);
+		m_menuSys.Insert(1, MenuId_Boot_CDVD2);
+		cdvd2 = menubar.FindItem(MenuId_Boot_CDVD2);
```

Comment:
Asserts in debug mode.

`Prepend` doesn't do anything useful, and `Insert` returns the inserted wxMenuItem, so the statement afterwards is somewhat useless.